test/fuzz/fuzz_dash_e.cc

```diff
@@ -33,6 +33,7 @@ unique_ptr<core::GlobalState> buildInitialGlobalState() {
     logger->trace("Doing on-start initialization");
 
     payload::createInitialGlobalState(gs, *opts, kvstore);
+    gs->runningUnderFuzzer = true;
```

Comment:
I believe we already have a way to check whether it's for the fuzzer. Let me check

Reply:
https://github.com/sorbet/sorbet/blob/4d55e84f9c26222e89c18609724f39ca31d960c8/common/common.h#L51-L55